test.js

```diff
@@ -25,10 +25,86 @@ test('number option', validate, {
 	number: ['score']
 });
 
-test('array option', validate, {
+test('default array option', validate, {
 	arr: 'array'
 }, {
-	array: ['arr']
+	array: [{key: 'arr', string: true}]
+});
+
+test('string array option', validate, {
+	arr: ['string']
+}, {
+	array: [{key: 'arr', string: true}]
+});
+
+test('number array option', validate, {
+	arr: ['number']
+}, {
+	array: [{key: 'arr', number: true}]
+});
+
+test('boolean array option', validate, {
+	arr: ['boolean']
+}, {
+	array: [{key: 'arr', boolean: true}]
+});
+
+test('multiple array options', validate, {
+	xs: ['number'],
+	ys: ['boolean']
+}, {
+	array: [{key: 'xs', number: true}, {key: 'ys', boolean: true}]
+});
+
+test('unsupported array element type fails', t => {
+	const error = t.throws(() => {
+		minimistOptions({
+			arr: {
+				type: ['date']
+			}
+		});
+	}, TypeError);
+
+	t.is(error.message, 'Expected "arr" to be one of [["string"], ["boolean"], ["number"]], got ["date"]');
```

Comment:
Should this be `Expected type of "arr" to be one of ...`?

Reply:
I just followed the naming style of the normal, non-array error message [here](https://github.com/vadimdemedes/minimist-options/blob/d920dff0a67f684d9adaa85c20745e2f152baa05/test.js#L245). If you want me to change it, I suggest we change the other one as well, to be consistent.

Reply:
Went ahead and changed both